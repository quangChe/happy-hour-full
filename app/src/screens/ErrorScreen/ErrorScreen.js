import React from 'react';

const ErrorPage = (props) => {
  return (
    <div style={{
      display: 'flex', 
      alignItems: 'center',
      justifyContent: 'center',
      fontSize: '3rem',
      fontWeight: 700,
      textAlign: 'center',
      color: '#e6e6e6'}}>404<br/>Page Not Found</div>
  )
}

export default ErrorPage;