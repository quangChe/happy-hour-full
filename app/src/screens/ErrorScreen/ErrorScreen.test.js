import React from 'react';
import { shallow } from 'enzyme';
import ErrorScreen from './ErrorScreen';

describe('Error Screen', () => {
  const screen = shallow(<ErrorScreen />);

  it('renders without crashing', () => {
    expect(screen).toMatchSnapshot();
  });
})