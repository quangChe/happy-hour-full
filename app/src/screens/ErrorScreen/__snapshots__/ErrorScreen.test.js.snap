// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Screen renders without crashing 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "color": "#e6e6e6",
      "display": "flex",
      "fontSize": "3rem",
      "fontWeight": 700,
      "justifyContent": "center",
      "textAlign": "center",
    }
  }
>
  404
  <br />
  Page Not Found
</div>
`;
